frameworks/helloworld/tests/test_recovery.py

```diff
@@ -90,19 +90,13 @@ def test_pod_stop():
     jsonobj = sdk_cmd.svc_cli(config.PACKAGE_NAME, config.SERVICE_NAME, 'pod info hello-0', json=True)
     assert len(jsonobj) == 1
     assert old_agent == jsonobj[0]['info']['slaveId']['value']
-    # TODO: this check should fail once command updates for stopped pods is implemented:
-    #       the task command should be updated to some new value that all stopped tasks would have
-    #assert 'stopped-task-cmd-here?' == jsonobj[0]['info']['command']['value']
-    assert old_cmd == jsonobj[0]['info']['command']['value'] # remove this and uncomment/update above
+    assert 'echo This task is PAUSED, sleeping ... && ./bootstrap --resolve=false && while true; do sleep 1209600; done' == jsonobj[0]['info']['command']['value']
```

Comment:
Feel free to just do a contains() or similar if this is too onerous. Just trying to validate the command has indeed been changed.

If you want, could also check that the health/readiness checks were updated as expected.

Reply:
Yes, this type of comparison seem pretty brittle and dependent on whitespace and general formatting.

One suggestion would be to split on `&&` and use `contains()` on each of the three expected parts. The idea of checking for a changed command is also a good option.